test/v2/services/RetrieveUkPropertyPeriodSummaryServiceSpec.scala

```diff
@@ -83,7 +83,7 @@ class RetrieveUkPropertyPeriodSummaryServiceSpec extends UnitSpec with ResponseM
           await(service.retrieveUkProperty(requestData)) shouldBe Left(ErrorWrapper(correlationId, error))
         }
 
-      val input = Seq(
+      val ifsErrorMap = Seq(
```

Comment:
We should avoid direct mention of `des` or `ifs` and use the more general `downstream` (i.e. on `line 78` update `ifsErrorCode` -> `downstreamErrorCode`)

The `ifsErrorMap` on `line 86` can be updated to `errorMap`